servant-server/test/Servant/ServerSpec.hs

```diff
@@ -314,7 +316,7 @@ postSpec = do
 type PutApi =
        ReqBody '[JSON] Person :> Put '[JSON] Integer
   :<|> "bla" :> ReqBody '[JSON] Person :> Put '[JSON] Integer
-  :<|> "empty" :> Put '[] ()
+  :<|> "empty" :> Put '[JSON] ()
```

Comment:
Actually, should I consolidate all the tests as well? I don't think it any longer makes sense to have separate tests for each method, since the code paths are all the same.